.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -519,7 +519,8 @@ jobs:
         if: steps.performance.outputs.is_dropped == 'true' && steps.issue-exists.outputs.result == 'false'
         uses: actions/github-script@a3e7071a34d7e1f219a8a4de9a5e0a34d1ee1293  # v4
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # do not use GITHUB_TOKEN here because it wouldn't trigger subsequent workflows
+          github-token: ${{ secrets.RASABOT_GITHUB_TOKEN }}
```

Comment:
do we have documentation somewhere about the permissions granted to the token? here I'm assuming that this token can create issues